rules/rule-manual/rule.metadata.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "rule-manual",
+    "attributes": {
+      "armoBuiltin": true,
+      "actionRequired": "manual review",
+      "hostSensorRule": false,
+      "imageScanRelated": false,
+      "controlConfigInputs": ""
```

Comment:
`controlConfigInputs` is a rule field (not under attributes) so can be deleted form the attributes section